Headphone Landing Page

itzwow opened the pull request:
# Description

Added a new Template "Headphone Landing Page"

Fixes:  #861 

<!---give the issue number you fixed----->

## Type of change

<!----Please delete options that are not relevant.And in order to tick the check box just but x inside them for example [x] like this----->

-
- [x ] New feature (non-breaking change which adds functionality)


# Checklist:

<!----Please delete options that are not relevant.And in order to tick the check box just but x inside them for example [x] like this----->
- [ x] I have made this from my own 
- [x ] My code follows the style guidelines of this project
- [ x] I have performed a self-review of my own code
- [x ] I have made corresponding changes to the documentation
- [ x] My changes generate no new warnings


# ATTACH SCREEN-SHOTS / DEPLOYMENT LINK
![preview](https://user-images.githubusercontent.com/84179065/169667288-528a03ae-1ced-40c3-ae2c-dede877cd03c.jpg)

VidhiBhatt01 commented on Headphone-Landing-Page/index.html:
```diff
@@ -0,0 +1,98 @@
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
```
Add proper title

VidhiBhatt01 reviewed (changes requested):
make it responsive and remove extra white space from the side
![image](https://user-images.githubusercontent.com/83345228/169677928-1c510971-8fdd-4e40-9012-e2be7774e2c5.png)

VidhiBhatt01 commented:
https://deploy-preview-866--cool-front-end-templates.netlify.app/headphone-landing-page/

itzwow commented:
I have updated the code and made it responsive
![12](https://user-images.githubusercontent.com/84179065/169768019-e095d637-712f-4245-bcf7-233c324f4be3.JPG)
![11](https://user-images.githubusercontent.com/84179065/169768032-cf12176b-5ea0-46cf-8bd8-38e3f0e330bb.JPG)

No more white space in the side 
Changed the title of the page:

![13](https://user-images.githubusercontent.com/84179065/169768278-3e532bca-e332-4e29-95bf-7f08fb2e8eee.JPG)

VidhiBhatt01 reviewed (changes requested):
Footer alignment in not proper and now some white space is added at the end
![image](https://user-images.githubusercontent.com/83345228/169811440-bd2d9d76-142a-4712-8078-5cbd10bbea64.png)

itzwow commented:
> Footer alignment in not proper and now some white space is added at the end ![image](https://user-images.githubusercontent.com/83345228/169811440-bd2d9d76-142a-4712-8078-5cbd10bbea64.png)

Actually the social icons are designed such way only to keep it at a fixed position.

The color issue has been resolved
![is](https://user-images.githubusercontent.com/84179065/169847168-97c1f510-d1d6-46ec-80bb-6a0c38053b53.JPG)
d
